src/api-calls.js

```diff
@@ -2,8 +2,13 @@ import { setData } from "./scripts";
 
 const getData = (data) => {
   return fetch(`http://localhost:3001/api/v1/${data}`)
-      .then(response => response.json())
-      .catch(error => console.log("ERROR", error));
+      .then(response => errorHandle(response))
+      .catch(error => 
+        {
+          console.log(error)
+          console.dir(error)
```

Comment:
curious as to why you chose these two instead of `console.error(err)`?

Reply:
No use, this was there for troubleshooting when I was trying to throw an error. Lines removed